WordPress/Classes/ViewRelated/Post/PostPreviewGenerator.swift

```diff
@@ -16,27 +16,20 @@ class PostPreviewGenerator: NSObject {
     }
 
     func generate() {
-        guard let link = post.permaLink,
-            let url = URL(string: link),
+        guard let url = post.permaLink.flatMap(URL.init(string:)),
             !post.hasLocalChanges() else {
-                showFakePreview()
-                return
+                return showFakePreview()
```

Comment:
Agreed. I found it a bit annoying to have the two lines everywhere, but I see how it can be way more confusing to `return Void`